CIP-0067/README.md

```diff
@@ -0,0 +1,53 @@
+---
+CIP:
+Title: Asset Name Label Registry
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Comments-URI:
+Status: Draft
+Type: Informational
+Created: 2022-07-13
+Post-History:
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a standard to classify Cardano native assets by the asset name.
+
+## Motivation
+
+As more assets are minted and different standards emerge to query data for these assets, it's getting harder for 3rd parties to determine the asset type and how to proceed with it. This standard is similar to [CIP-0010](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0010), but focuses on the asset name of an asset.
+
+
+## Specification
+
+To classify assets the `asset_name` needs to be prefixed with an opening and closing parentheses and the label in between: `({Label})`.
+ 
+For example:
+
+UTF-8 encoded: `(123)TestToken`\
+HEX encoded: `283132332954657374546F6B656E`
+
+
+These are the reserved `asset_name_label` values
+
+`asset_name_label`            | description
+----------------------------  | -----------------------
+0 - 15                        | reserved\*
+65536 - 131071                | reserved - private use
+
+For the registry itself, please see [registry.json](./registry.json) in the machine-readable format. Please open your pull request against
+this file.
```

Comment:
One problem we had / have with CIP-0010 is that there's no particular "rule" that defines what can go in the registry. As editors we try to do some basic sanity check and ask people to pitch / justify a bit their project; but it would be nice / preferable if these rules were specified in the specification itself. For example: 

---

### Adding an entry to the registry

To propose an addition to the registry edit the `registry.json` with your details, open a pull request against the CIPs repository and give a brief description of your project and how you intend to use metadata associated with the label entry.